### ISSUE TITLE
Progress Bars For Loading and Export

### ISSUE THREAD

**Issue by ds5678** [AssetRipper/AssetRipper#1348]

**Comment by leumasme** [comment]
Agree. Missing progress indicator for initially opening the Folder was confusing on my first use.  
Even console output stopped for a while, so the only indicator that "Open Folder" actually did anything was high CPU usage; so even just replacing the "No files loaded" text after selecting files/folder would be a better experience (instead of aiming for a fully dynamic progress bar as the first goal).
